launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -48,6 +48,46 @@ void CModListView::changeEvent(QEvent *event)
 	QWidget::changeEvent(event);
 }
 
+void CModListView::dragEnterEvent(QDragEnterEvent* event)
+{
+	event->acceptProposedAction();
+}
+ 
+void CModListView::dragMoveEvent(QDragMoveEvent* event)
+{
+	event->acceptProposedAction();
+}
+ 
+void CModListView::dragLeaveEvent(QDragLeaveEvent* event)
+{
+	event->accept();
+}
```

Comment:
Iirc these are not required to be implemented

Reply:
dragEnterEvent is needed